server/app/services/applicant/ApplicantService.java

```diff
@@ -1071,6 +1097,10 @@ private void logDuplicateDrafts(ImmutableSet<Application> applications) {
   public abstract static class ApplicantProgramData {
     public abstract ProgramDefinition program();
 
+    /**
+     * Returns whether an application is eligible for a program, and empty if there are no
+     * eligibility conditions for the program.
+     */
```

Comment:
**suggestion (blocking):** clarify what this does for incomplete eligibility criteria i.e. the applicant hasn't answered all (or any) eligibility questions for a program

Reply:
Done